src/analysis/calculus/bump_function_inner.lean

```diff
@@ -236,6 +236,16 @@ zero_of_nonpos le_rfl
 @[simp] protected lemma one : smooth_transition 1 = 1 :=
 one_of_one_le le_rfl
 
+/-- Since `real.smooth_transition` is constant on $(-∞, 0]$ and $[1, ∞)$, applying it to the projection
+of `x : ℝ` to $[0, 1]$ gives the same result as applying it to `x`. -/
```

Comment:
```suggestion
/-- Since `real.smooth_transition` is constant on $(-∞, 0]$ and $[1, ∞)$, applying it to the
projection of `x : ℝ` to $[0, 1]$ gives the same result as applying it to `x`. -/
```